index.html

```diff
@@ -1,189 +1,227 @@
+                    users to add to an existing story over the course of a day without spamming their friends.
+                </p>
+                <ul class="trainning">
+                    <li class="other-skills">html</li>
+                    <li class="ruby">Ruby on rails</li>
+                    <li class="other-skills">css</li>
+                    <li class="other-skills">javascript</li>
+                </ul>
+                <button class="card-btn">See Project</button>
+            </div>
+            </div>
+            <div class="card">
+                <div class="three">
+                <img  src="1.png" alt="">
+            </div>
+            <div>
+                <h2 class="title">Facebook 360</h2>
+            
+                    <ul class="field">
+                        <li class="cano">FACEBOOK</li>
+                        <li class="tag">&#8226; Full Stack Dev </li>
+                        <li class="tag">&#8226; 2015</li>
+                    </ul>
+                    <p class="instruction">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </p>
+                <p class="desk-praa">
+                    Exploring the future of media in Facebook's first Virtual Reality app; a place
+                     to discover and enjoy 360 photos and videos on Gear VR.
+                    </p>
+                <ul class="trainning">
+                    <li class="other-skills">html</li>
+                    <li class="ruby">Ruby on rails</li>
+                    <li class="other-skills">css</li>
+                    <li class="other-skills">javascript</li>
+                </ul>
+                <button class="card-btn">See Project</button>
+            </div>
+        </div>
+            <div id="for" class="card">
+                <div class="pic">
+                    <img   class="pic" src="2.png" alt="">
+                </div>
+            <div>
+                <h2 class="title">Uber Navigation</h2>
+                    <ul class="field">
+                        <li class="cano">Uber</li>
+                        <li class="tag">&#8226; Lead Developer </li>
+                        <li class="tag">&#8226; 2018</li>
+                    </ul>
+                    <p class="instruction">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </p>
+                <p class="desk-praa"> 
+                    A smart assistant to make driving more safe, 
+                    efficient, and fun by unlocking your most expensive computer: your car.
+                </p>
+                <ul class="trainning">
+                    <li class="other-skills">html</li>
+                    <li class="ruby">Ruby on rails</li>
+                    <li class="other-skills">css</li>
+                    <li class="other-skills">javascript</li>
```

Comment:
- [ ] The skills listed in each of the cards on the figma design are aligned horizontally and well spaced. You can see how they give a very nice visual view and will  definitely draw the attention of a viewer. E.g a hiring manager.
- Kindly adjust yours in that regard. `flexbox` will make implementing that functionality very seamless. 

## Your work:
![image](https://github.com/mumbereh/my-portfolio/assets/50990925/aa903ae0-f870-4058-9bab-e3be22e8492f)

## Figma deisgn:
![image](https://github.com/mumbereh/my-portfolio/assets/50990925/c3e85599-6d50-4ddd-9e99-754f245be891)